<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QWidget" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>301</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="cBoxDisplayTC">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Display timecode</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="cBoxDisplayNextTC">
           <property name="text">
            <string>Display next timecode</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="cBoxDisplayNextText">
           <property name="text">
            <string>Display next texts</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="cBoxDisplayTitle">
           <property name="text">
            <string>Display title</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QCheckBox" name="cBoxDisplayLoop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Display loop number</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="cboBoxLang"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabStrip">
       <attribute name="title">
        <string>Strip</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="layoutStrip">
          <item row="2" column="0">
           <layout class="QVBoxLayout" name="layoutFont">
            <item>
             <widget class="QLineEdit" name="lineEditFilter">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetFont"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QFormLayout" name="layoutBoldness">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::FieldsStayAtSizeHint</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblBoldness">
              <property name="text">
               <string>Bold :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="sliderBoldness">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1" alignment="Qt::AlignHCenter">
           <widget class="QSlider" name="sliderStripHeight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblStripHeight">
            <property name="text">
             <string>Strip Height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QFormLayout" name="layoutItalic">
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="lblItalic">
              <property name="text">
               <string>Italic :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="sliderItalic">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QFormLayout" name="layoutForm">
            <item row="0" column="0">
             <widget class="QLabel" name="lblSpeed">
              <property name="text">
               <string>Speed:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxSpeed">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>12</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSynchro">
       <attribute name="title">
        <string>Synchronisation</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblDelay">
            <property name="text">
             <string>Delay</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QListWidget" name="listWidgetSync">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>75</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>No sync</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sony 9 pin</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="radioButtonQF">
            <property name="text">
             <string>quarter frame</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignLeft">
           <widget class="PhLockableSpinBox" name="spinBoxDelay">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="wordWrap" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBoxSony">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>Parameters</string>
            </property>
            <widget class="QLabel" name="lblSonyHighSpeed">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>61</y>
               <width>114</width>
               <height>24</height>
              </rect>
             </property>
             <property name="text">
              <string>Sony high speed</string>
             </property>
            </widget>
            <widget class="QLabel" name="lblSonyID">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>114</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Sony ID</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinBoxSonyHighSpeed">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>61</y>
               <width>75</width>
               <height>24</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
            <widget class="QLabel" name="lblInputs">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>121</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>Available inputs</string>
             </property>
            </widget>
            <widget class="QListWidget" name="listWidgetInputs">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>30</y>
               <width>200</width>
               <height>100</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEditSonyID">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>30</y>
               <width>70</width>
               <height>21</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">F0 C0</string>
             </property>
             <property name="maxLength">
              <number>20</number>
             </property>
             <property name="placeholderText">
              <string notr="true"/>
             </property>
            </widget>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="radioButtonMS">
            <property name="text">
             <string>millisecond</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="PhDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PhDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>PhCommonUI/PhDialogButtonBox.h</header>
  </customwidget>
  <customwidget>
   <class>PhLockableSpinBox</class>
   <extends>QSpinBox</extends>
   <header>PhCommonUI/PhLockableSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
